community/modules/compute/gke-topology-scheduler/main.tf

```diff
@@ -0,0 +1,27 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+module "kubectl_apply" {
+  source = "../../../../modules/management/kubectl-apply"
+
+  cluster_id = var.cluster_id
+  project_id = var.project_id
+
+  apply_manifests = [
```

Comment:
Is there a way to verify that node pool has compact policies applied ?

Reply:
I'm not completely sure which nodepool?
We are not passing any node-pool module to this module.

Reply:
Ah okay , I thought topology scheduler only worked when we have nodes giving their placement information .  So if that is not found, topology scheduler is not helpful. But if that is not the case, feel free to disregard.

Reply:
Followed up offline, we will work on this toleration if it is required as part of blueprint that is using this topology scheduler

Reply:
Resolving this conversion. Will be tested with the xpk-a3 blueprints and modified if reuired.